spec/system/task_spec.rb

```diff
@@ -1,36 +1,38 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { FactoryBot.create(:project)}
```

Comment:
`FactoryBot.`の部分は定義を追記することで省略可能です。
全体のコード量が削減できて見通しが良くなるので省略していきましょう。
```suggestion
  let(:project) { create(:project) }
```

Reply:
修正いたしました。
a69978519093af896390a9f1900bac1472671732